code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -15,6 +13,8 @@
 /datum/reagent/medicine/on_mob_life(mob/living/carbon/M)
 	current_cycle++
 	holder.remove_reagent(type, metabolization_rate / M.metabolism_efficiency) //medicine reagents stay longer if you have a better metabolism
+	if(metabolite)
+		holder.add_reagent(metabolite, metabolization_rate / M.metabolism_efficiency * METABOLITE_RATE)
```

Comment:
That doesn't make sense though. 

Only three reagents are intended to have this as an overdose effect (at this time, at least) and they are not intended to follow the normal metabolite logic because they aren't being metabolized normally. Look at the handful of toxins that effectively produce histamine as a metabolite - the body views the reagent as foreign and is rushing to convert it. 

I do not want the overdose mechanic to match either of the two normal processing rates.

Reply:
maybe not necessary since you wanted the overdose part working separated